languages/translations/eo.json

```diff
@@ -422,53 +422,23 @@
         },
         "exit roundabout": {
             "default": {
-                "default": "Veturu {modifier}",
-                "name": "Veturu {modifier} al {way_name}",
-                "destination": "Veturu {modifier} direkte al {destination}"
-            },
-            "left": {
-                "default": "Turniĝu maldekstren",
-                "name": "Turniĝu maldekstren al {way_name}",
-                "destination": "Turniĝu maldekstren direkte al {destination}"
-            },
-            "right": {
-                "default": "Turniĝu dekstren",
-                "name": "Turniĝu dekstren al {way_name}",
-                "destination": "Turniĝu dekstren direkte al {destination}"
-            },
-            "straight": {
-                "default": "Veturu rekten",
-                "name": "Veturu rekten al {way_name}",
-                "destination": "Veturu rekten direkte al {destination}"
+                "default": "Elveturu trafikcirklegon",
+                "name": "Elveturu trafikcirklegon al {way_name}",
+                "destination": "Elveturu trafikcirklegon direkte al {destination}"
             }
         },
         "exit rotary": {
             "default": {
-                "default": "{modifier}",
-                "name": "{modifier} al {way_name}",
-                "destination": "Veturu {modifier} direkte al {destination}"
-            },
-            "left": {
-                "default": "Turniĝu maldekstren",
-                "name": "Turniĝu maldekstren al {way_name}",
-                "destination": "Turniĝu maldekstren direkte al {destination}"
-            },
-            "right": {
-                "default": "Turniĝu dekstren",
-                "name": "Turniĝu dekstren al {way_name}",
-                "destination": "Turniĝu dekstren direkte al {destination}"
-            },
-            "straight": {
-                "default": "Veturu rekten",
-                "name": "Veturu rekten al {way_name}",
-                "destination": "Veturu rekten direkte al {destination}"
+                "default": "Eliru trafikcirklegon",
+                "name": "Elveturu trafikcirklegon al {way_name}",
+                "destination": "Elveturu trafikcirklegon direkte al {destination}"
             }
         },
         "turn": {
             "default": {
-                "default": "Veturu {modifier}",
```

Comment:
This changes seems suspect to me. Can someone confirm that there is no verb needed to tell someone to turn in esperanto?

Reply:
Its also possible that the direction word is its own verb.

Reply:
In this localization, the `modifier` words are all translated into adverbs like “maldekstren” (left), which can’t technically be used on their own as imperative verbs. Colloquially, one can say “Left!” in English, and I’d imagine that’s true in most other languages too, including Esperanto. But I think the previous translation was more faithful to the original English. Perhaps @tcql can weigh in.

@verdulo originally put in “Veturu” (to go), which I think makes sense here, but recently removed it, perhaps because one of the direction words, `uturn`, was translated as “turniĝu malantaŭen”, so “Veturu turniĝu malantaŭen” would’ve been redundant. However, this particular set of strings (`turn.default.*`) is never used, because all the possible `modifier`s under `turn` have specific translations.

https://github.com/Project-OSRM/osrm-text-instructions/blob/5f78287907cf83c1495cc3a45f64fefe600b1dc8/test/fixtures/v5/turn/left_default.json#L12

Nor does the `uturn` modifier translation ever get used, because the `turn.uturn.*` strings don’t include `{modifier}`.

https://github.com/Project-OSRM/osrm-text-instructions/blob/5f78287907cf83c1495cc3a45f64fefe600b1dc8/test/fixtures/v5/turn/uturn_default.json#L12

For now, I’ve reverted that change in Transifex and on this branch.